engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -36,7 +36,7 @@ class CromwellApiServiceSpec extends AsyncFlatSpec with ScalatestRouteTest with
 
   implicit def default = RouteTestTimeout(5.seconds)
 
-  "REST ENGINE /stats endpoint" should "return 200 for stats" in {
+  "REST ENGINE /stats endpoint" should "return 200 for stats" ignore {
```

Comment:
IMHO the test should be updated to match the new expectation, and not leave this cruft behind. That is unless `/engine/v1/stats` is coming back, in which case there should be a link to the ticket with acceptance criteria.

That said, maybe a future PR can clean this up.